db/view/view.cc

```diff
@@ -1777,8 +1777,10 @@ future<> view_update_generator::mutate_MV(
                     mut.s->ks_name(), mut.s->cf_name(), base_token, view_token);
             local_view_update = _proxy.local().mutate_mv_locally(mut.s, *mut_ptr, tr_state, db::commitlog::force_sync::no).then_wrapped(
                     [s = mut.s, &stats, &cf_stats, tr_state, base_token, view_token, my_address, mut_ptr = std::move(mut_ptr),
-                            sem_units] (future<>&& f) {
+                            sem_units, this] (future<>&& f) mutable {
```

Comment:
Please correct me if I'm wrong, but isn't this code a **local** mutation, i.e., in this current node, which is always synchronous and doesn't increase the view update backlog? So why do you need to touch the backlog here?

Reply:
This doesn't increase the backlog only from the perspective of the client sending this request - the backlog does actually increase at the start of the process but it will always come back down before the request finishes, but in the meantime, the backlog will be higher for all other cilents.

If you were referring to the background tasks that only remote updates are generating, it's true that this branch does not create them, but it still holds on to some resources (in particular, to the mutation)